tests/framework/extensions/namespaces/getnamespaces.go

```diff
@@ -0,0 +1,23 @@
+package namespaces
+
+import (
+	"sort"
+
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+)
+
+// GetSortedNamespaces is a helper which returns a sorted list of namespace names
+func GetSortedNamespaces(steveclient *v1.Client) ([]string, error) {
```

Comment:
What do you think about renaming this to `ListNamespaceNames`? I think the naming would be more unified with the below project name list.

Reply:
yes, I think that would be appropriate instead of `get`.